File: backend-gin/internal/scripts/test_hf.go
package scripts

import (
	"backend/internal/lib/clients"
	"fmt"
)

func TestHuggingFace() {
	hf := clients.NewHuggingFaceClient()
	vec, err := hf.EmbedText("I am trying to debug my code in go")
	if err != nil {
		fmt.Println(err)
	} else {
		fmt.Println(vec)
	}
}

File: backend-express/src/scripts/testHf.ts
import { huggingFace } from "@/lib/clients";
import { closeAll, startAll } from "@/lib/setup";

async function test() {
    const result = await huggingFace.embedText(
        "I am trying to debug my code in go"
    );
    console.log(result);
}

if (require.main === module) {
    startAll().then(test).finally(closeAll);
}
